src/bp/admin/ui/src/workspace/bots/BotItemCompact.tsx

```diff
@@ -45,10 +46,22 @@ const BotItemCompact: FC<Props> = ({
   reloadBot,
   viewLogs
 }) => {
+  const [supportedLanguages, setSupportedLanguages] = useState([])
+  useEffect(() => {
+    const axios = api.getSecured({ useV1: true })
+
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    axios.get(`/bots/${bot.id}/mod/nlu/health`).then(response => {
+      setSupportedLanguages(response.data)
```

Comment:
Don't forget to fetch the values from the `validLanguages` property of the response. 

Also, since the call will return the same value for all bots, you should create an action and put the value in the redux store. You can use the triple underscores ("___") bot ID which is a wildcard for "any bot".